packages/tslint-config/src/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
     "extends": "../../../config/tsconfig.base",
     "compilerOptions": {
+        "lib": ["es6"],
         "module": "commonjs",
-        "outDir": "../lib/rules"
+        "outDir": "../lib/rules",
+        "skipLibCheck": true
```

Comment:
why?

Reply:
Previously, I didn't include the dom library, which led to some of the @types types in the root node_modules directory being angry. I've now added the dom library, even though it's unused, since I think that seems like the lesser of two evils.